keps/sig-scheduling/4815-dra-partitionable-devices/README.md

```diff
@@ -411,18 +437,19 @@ type ResourceSliceSpec struct {
   // must specify this individually.
   //
   // Exactly one of NodeName, NodeSelector, AllNodes, and PerDeviceNodeSelection
-  // must be set.
+  // must be set when the `Devices` field is set. If the `SharedCounters` field is
+  // set, none of the fields can be set.
   //
   // +optional
-  // +oneOf=NodeSelection
+  // +featureGate=DRAPartitionableDevices
   PerDeviceNodeSelection bool
 
   // SharedCounters defines a list of counter sets, each of which
   // has a name and a list of counters available.
   //
-  // The names of the SharedCounters must be unique in the ResourceSlice.
+  // The names of the Counter Sets must be unique in the ResourcePool.
   //
-  // The maximum number of counters in all sets is 32.
+  // The maximum number of counter sets is 8.
```

Comment:
Can I have multiple slices that define additional counter sets? 8 sounds too low pool-wide, otherwise.